ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantSessionConfiguration.scala

```diff
@@ -11,4 +11,6 @@ private[testtool] final case class ParticipantSessionConfiguration(
     port: Int,
     ssl: Option[TlsConfiguration],
     partyAllocation: PartyAllocationConfiguration,
-)
+) {
+  lazy val address: String = s"$host:$port"
```

Comment:
Is there a case in which you are not using `address` that makes laziness useful? The operation is also quite light.

Reply:
100% agree.